deployment_report/model/viya_deployment_report.py

```diff
@@ -504,6 +522,18 @@ def get_sas_component_resources(self, component_name: Text, resource_kind: Text)
         except KeyError:
             return None
 
+    def get_cadence_data(self, data: Dict) -> Optional[Dict]:
```

Comment:
New methods need to have pytest coverage by implementing a test_get_cadence_data() in deployment_report/test/test_deployment_report.py.     Work with @sasjowood on this.

Reply:
test added with new code.

Reply:
Looks good.  Thank you!